cartridges/int_alma/cartridge/scripts/steps/CheckRefund.js

```diff
@@ -45,14 +31,15 @@ exports.execute = function () {
     var Status = require('dw/system/Status');
     var orders = getOrdersRefunded();
     var errors = [];
+    var refundHelper = require('*/cartridge/scripts/helpers/almaRefundHelper');
 
     Logger.info('[INFO][ALMA refund] job launched for: ' + orders.count + ' orders.');
     if (orders.count > 0) {
         while (orders.hasNext()) {
             var orderItem = orders.next();
             if (isOrderToBeRefund(orderItem)) {
                 try {
-                    refundPaymentForOrder(orderItem);
+                    refundHelper.refundPaymentForOrder(orderItem, orderItem.custom.ALMA_Wanted_Refund_Amount);
```

Comment:
if everything went well, we should reset `ALMA_wanted_refund_amount`  to 0 as well